app/build.gradle

```diff
@@ -6,6 +6,7 @@ apply plugin: 'kotlin-kapt'
 
 android {
   compileSdkVersion deps.android.build.compileSdkVersion
+  buildToolsVersion '28.0.2'
```

Comment:
Why is this needed?

Reply:
without it, it's defaulting to this version anyway (forced by the Android gradle plugin i think?) - so figured might as well set it explicitly so it's not resolving to something different without us knowing what it is.

Reply:
That's right, the tooling version is selected automatically depending on the Android Gradle version. [The prior change](https://github.com/quran/quran_android/pull/946) was intentional :)